src/piral-new.test.ts

```diff
@@ -61,9 +61,42 @@ runTests('piral-new', ({ test }) => {
     },
   );
 
+  test(
+    'from-cli-moved',
+    'can create a new TS piral instance with modules using `piral-core` from cli in a new directory',
+    [],
+    async (ctx) => {
+      await ctx.run(
+        `npx --package piral-cli@${cliVersion} piral new --tag ${cliVersion} --framework piral-core --base my-piral-instance`,
+      );
+
+      await ctx.assertFiles({
+        'my-piral-instance/package.json'(content) {
+          expect(content).not.toBe('');
+          expect(content).toContain('"typescript"');
+          expect(content).toContain('"piral-core"');
+          expect(content).toContain('"piral-cli"');
+          expect(content).toContain(`"my-piral-instance"`);
+        },
+        'my-piral-instance/tsconfig.json': true,
+        'my-piral-instance/node_modules/piral-cli/package.json': true,
+        'my-piral-instance/src/index.html'(content) {
+          expect(content).not.toBe('');
+          expect(content).toContain('<script src="./index.tsx"></script>');
+        },
+        'my-piral-instance/src/index.tsx'(content) {
+          expect(content).not.toBe('');
+          expect(content).toContain('from "piral-core"');
+          expect(content).toContain('fetch("https://feed.piral.cloud/api/v1/pilet/empty")');
+          expect(content).toContain('render(<Piral instance={instance} />, document.querySelector("#app"));');
+        },
+      });
+    },
+  );
+
   test(
     'from-cli-invalid-template',
-    'can not create a new TS piral instance without modules using invalid template from cli in the same directory',
+    'can create a new TS piral instance without modules using invalid template from cli in the same directory',
```

Comment:
This needs to remain as "can not". We are using a try-catch block and expect the CLI to error here.